### PR TITLE
Jyan/oss release notes 2020 10

### PR DESCRIPTION
**Pull request by jyanJing**
2020.10.0 Release Notes

### CODE REVIEW

**Review by yufangong — APPROVED** [review]

File: source/blog/2020-10-27-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+---
+layout: post
+title: October 2020 Release Notes - Version 20.10.0 🎃
+published: true
+post_author:
+  display_name: Jing Yan
+  twitter: jyan
```
Comment (ryanoneill) [inline source/blog/2020-10-27-release-notes.md:7]: Replace with 'freshlemonfish'. 😃

File: source/blog/2020-10-27-release-notes.md [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+---
+
+Trick or... our October release is as sweet as candies 👻
+
+### [Finagle](https://github.com/twitter/finagle/) ###
+
+#### Breaking API Changes
+
+-   finagle-thrift: Change the partition locator function getLogicalPartitionId in
+    PartitioningStrategy from Int =&gt; Int to Int =&gt; Seq\[Int\], which supports many to many mapping
```
Comment (ryanoneill) [inline source/blog/2020-10-27-release-notes.md:18]: No idea whether the rendering of this line is correct, but just a chance to double check.

**Review by ryanoneill — APPROVED** [review]

File: source/blog/2020-10-27-release-notes.md (hunk 1 above)
Comment (jyanJing) [reply source/blog/2020-10-27-release-notes.md]: updated, for a second I thought it was a joke 😅

File: source/blog/2020-10-27-release-notes.md (hunk 2 above)
Comment (jyanJing) [reply source/blog/2020-10-27-release-notes.md]: good catch, updated!

**Pull request merged**